src/commonMain/kotlin/com/algolia/search/model/places/PlaceMulti.kt

```diff
@@ -0,0 +1,56 @@
+package com.algolia.search.model.places
+
+import com.algolia.search.model.ObjectID
+import com.algolia.search.model.search.Point
+import com.algolia.search.model.search.Language
+import com.algolia.search.model.search.RankingInfo
+import com.algolia.search.serialize.*
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.Transient
+import kotlinx.serialization.json.JsonObject
+
+
+@Serializable
+data class PlaceMulti(
+    @SerialName(KeyCity) val cityOrNull: Map<Language, List<String>>? = null,
```

Comment:
Could make it more maintenable/better internal DX by having the same param ordering between `PlaceMono` and `PlaceMulti`

Reply:
Agreed